frontend/src/routes/SignUp/index.js

```diff
@@ -95,6 +121,11 @@ export default function SignUp(props) {
               name="email"
               autoComplete="email"
               onChange={handleFormChange("email")}
+              helperText={
+                formValues.email.length > 0 && !validateEmail(formValues.email)
+                  ? "Please enter a valid email."
+                  : null
+              }
```

Comment:
Duplicate error message. Please delete one of them.
![image](https://user-images.githubusercontent.com/37783082/89865683-07f58e80-db63-11ea-9648-d4c36f0646e9.png)